stonesoup/metricgenerator/plotter.py

```diff
@@ -46,92 +42,60 @@ def compute_metric(self, manager, *args, **kwargs):
         return metric
 
     def plot_tracks_truth_detections(self, tracks, groundtruth_paths,
-                                     detections):
+                                     detections, uncertainty=False, particle=False):
```

Comment:
So `uncertainty` and `particle` should be `Property`s on the class as well, passed when calling this method from `TwoDPlotter.compute_metric()`.